numpy/fft/pocketfft.py

```diff
@@ -44,7 +44,11 @@
     overrides.array_function_dispatch, module='numpy.fft')
 
 
-def _raw_fft(a, n, axis, is_real, is_forward, fct):
+# `inv_norm` is a float by which the result of the transform needs to be
```

Comment:
I'm wondering if this should just be called "norm" - at least in the non-fft contexts I've seen norm used, it's the thing you divide by to produce something unitary. Don't change it without someone with more domain knowledge following up though.

Reply:
I think `norm` technically makes a little more sense than `inv_norm` for the reasons described, but I don't think it is too big of a deal as this variable doesn't appear in the API.